src/bosh-director/lib/bosh/blobstore_client/azurestoragecli_blobstore_client.rb

```diff
@@ -0,0 +1,138 @@
+require 'openssl'
+require 'digest/sha1'
+require 'base64'
+require 'securerandom'
+require 'open3'
+require 'json'
+
+module Bosh::Blobstore
+  class AzurestoragecliBlobstoreClient < BaseClient
```

Comment:
Shouldn't that be real camel case?  `AzureStorageCliBlobstoreClient`

Reply:
No it is not possible, because the class will be load [here](https://github.com/cloudfoundry/bosh/blob/c734396373987b0834a63de4e1b9c6ff0e5cac2b/src/bosh-director/lib/bosh/blobstore_client/client.rb#L25). Only the first character will be upper-case. That's also why in [s3cli](https://github.com/cloudfoundry/bosh/blob/c734396373987b0834a63de4e1b9c6ff0e5cac2b/src/bosh-director/lib/bosh/blobstore_client/s3cli_blobstore_client.rb#L9) 'c' is not capitalised.

Reply:
Ah oke